src/textmate/bicep.tmlanguage

```diff
@@ -216,7 +216,7 @@
         <key>name</key>
         <string>keyword.control.declaration.bicep</string>
         <key>match</key>
-        <string>\b(metadata|targetScope|resource|module|param|var|output|for|in|if|existing|import|as|type|with|using|func|assert|provider)\b</string>
```

Comment:
Could you also update https://github.com/Azure/bicep/blob/main/docs/grammar.md

Reply:
as we discussed on the sync call I mark this comment as resolved 👍

Reply:
as we discussed offline, I'm marking this as resolved 👍